### ISSUE TITLE
Growl 1.3 Compatibility

### ISSUE THREAD

**Issue by SBejga** [chamerling/JavaGrowl#1]
did you ever tried with Growl 1.3? Your class does not work with my current growl. (Mac OS X Lion). Unfortunately.

best regards.

**Comment by chamerling** [comment]
No, I did not buy it so I did not test this compatibility, sorry
